### PR TITLE
Proposed README tweaks

### PR DESCRIPTION
**Pull request by it33**
Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).

When filling in a section please remove the help text and the above text.

#### Summary
Proposed README tweaks

#### Ticket Link
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,48 @@
...
 
-- **Contribute Code** - View our [Code Contribution Guidelines](http://docs.mattermost.com/developer/contribution-guide.html)
-- **Work on "Help Wanted" projects** - See a list of [tickets accepting pull requests](https://mattermost.atlassian.net/issues/?filter=10101)
-- **Discuss Contributions** - Join our [Developer Discussion](https://pre-release.mattermost.com) for contributors on the nightly builds server
-- **File Bugs** - Search for existing bugs and [file a GitHub issue if your bug is new](http://www.mattermost.org/filing-issues/)
-- **Share Feature Ideas** - Add a [feature idea to be discussed or upvoted](http://www.mattermost.org/feature-requests/)
-- **Get Troubleshooting Help** - Search articles and [add a topic if the issue is new](https://forum.mattermost.org/t/how-to-use-the-troubleshooting-forum/150)
+- [Contribute Code](http://docs.mattermost.com/developer/contribution-guide.html)
+- [Find "Help Wanted" projects](https://mattermost.atlassian.net/issues/?filter=10101)
+- [Join Developer Discussion on a Mattermost Server for contributors](https://pre-release.mattermost.com)
```
Comment (jwilander) [inline README.md]: Maybe make this the team invite link?

**Comment by jwilander** [comment]
+1 other than one comment

**Comment by coreyhulen** [comment]
+1 from me, but @it33 you have a comment from joram.

**Comment by lfbrock** [comment]
+1, and I updated the link

**Comment by jwilander** [comment]
+1

**Pull request merged**